core/filenumber_checker.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from pathlib import Path
+
+import pandas as pd
+from moviepy.video.io.VideoFileClip import VideoFileClip
+
+from utils import check_if_same, convert_to_path
+
+class Framenumberchecker():
+    """Checks if framenumbers of all videos are the same
+    Input: List of videofilepaths
+    """
+    def check_framenumber(self, videofilepaths: List) -> bool:
+        """checks if all videos in videofilepaths list have the same number of frames"""
+        video_framenumbers = []
+        for elem in videofilepaths:
+            video = Framenumberreader()
+            framenumber = video.run(convert_to_path(elem))
+            video_framenumbers.append(framenumber)
+        if check_if_same(video_framenumbers):
+            print('All videos have the same amount of frames!')
+        else:
+            print('Not all videos have the same number of frames!')
+            pd.DataFrame(video_framenumbers, index = [videofilepaths])
```

Comment:
What's the intention behind creating this DataFrame?

Reply:
showing which videos have different frame numbers. In the next step we could extract the video that is faulty and fix/get around the issue

Reply:
So, probably you want to save this DataFrame as a variable to make it accessible later on :D